### ISSUE TITLE
Couple of issues (with solution)

### ISSUE THREAD

**Issue by aferdous** [michaelklishin/cassandra-chef-cookbook#199]
Hi,

1. Kitchen - inside .kitchen.yaml - recipe name cassandra had to be replaced with cassandra-dse to make it work. 
2. In the recipes/datastax.rb there was a missing line of creating the /etc/cassandra folder for debian installation. I had to add the line  node['cassandra']['conf_dir'] to fix this.

[node['cassandra']['installation_dir'],
 node['cassandra']['conf_dir'],
 node['cassandra']['bin_dir'],
 node['cassandra']['log_dir'],
 node['cassandra']['root_dir'],
 node['cassandra']['lib_dir']
].each do |dir|
  directory dir do
    owner node['cassandra']['user']
    group node['cassandra']['group']
    recursive true
    mode 0755
  end
end

Thought I would share it with you guys.

Ahmed

**Comment by michaelklishin** [comment]
@aferdous please submit a pull request ;)

**Comment by aferdous** [comment]
Hi Michael - I saw you have already fixed the kitchen.yaml file. I just submitted a pull request for the recipes/datastax.rb to fix the conf folder creation issue. 

Other than this - I just wanted to say thank you and everyone that contributed to development of this cookbook.  We have just setup a cluster using the current version to develop a proof of concept and apart from these couple of small issues - it worked as expected. the README.md says its not production ready yet (e.g. differentiating between seed nodes and a non-seed node etc. I had to manually change the attributes to the ip address for seed node - but I can live with that for now) but I am sure it will reach there soon. From my side, I will keep posting (and fixing if I can) if we find anything else here. 

Cheers,

Ahmed
